edit rules.md

SeoJin0513 opened the pull request.

SeoJin0513 committed:
edit rules.md

rules.md 룰 수정

kimch0612 commented on documents/rules.md:
```diff
@@ -12,15 +12,57 @@
-- Choice: 
-- 4 Kind of: 
-- Full House: 
+#### Upper Section에서의 조건
```
~~에서의 조건에서 작성한 이름을 [Manual.md](https://github.com/kimch0612/OOP2_Project/blob/master/documents/manual.md) 파일과 동일하게 작성해주세요.

kimch0612 commented on documents/rules.md:
```diff
@@ -12,15 +12,57 @@
+- Sixes: 6이 나온 주사위의 개수와 합계 점수
+- Upper Section Bonus: 상위 섹션의 점수 합계가 63 이상일 경우 35점의 보너스를 획득.
+#### Lower Section에서의 조건
+- Three of a Kind: 세 개 이상의 동일한 숫자를 얻으면 모든 주사위의 합계.
```
Three of a Kind는 우리 게임에서 존재하지 않는 룰 입니다.

kimch0612 commented on documents/rules.md:
```diff
@@ -12,15 +12,57 @@
+- Upper Section Bonus: 상위 섹션의 점수 합계가 63 이상일 경우 35점의 보너스를 획득.
+#### Lower Section에서의 조건
+- Three of a Kind: 세 개 이상의 동일한 숫자를 얻으면 모든 주사위의 합계.
+- Four of a Kind: 네 개 이상의 동일한 숫자를 얻으면 모든 주사위의 합계.
```
Four of a Kind와 Small & Large Straight의 이름을 게임에서 사용한 이름과 동일하게 작성해주세요.

kimch0612 commented on documents/rules.md:
```diff
@@ -12,15 +12,57 @@
+- Small Straight: 4개의 연속된 숫자를 얻으면 30점.
+- Large Straight: 5개의 연속된 숫자를 얻으면 40점.
+- Yahtzee: 다섯 개의 동일한 숫자를 얻으면 50점. 다시 야찌를 얻으면 보너스로 100점을 획득할 수 있음.
+- Chance: 어떤 숫자든 상관 없이 주사위의 합계.
```
Choice는 C-Y Dice Section에서 맨 위에 위치하게끔 위치를 조정해주세요.

kimch0612 reviewed (changes requested):
아래의 수정 사항을 적용 후 답변 바랍니다.
1. 작업 협약에 의거, PR의 Summary와 Description을 작성해주세요.
2. Request changes의 내용을 확인 후 수정해주세요.

kimch0612 commented:
[Issues](https://github.com/kimch0612/OOP2_Project/issues/18)에 작성한 내용도 프로젝트 협약에 의거하여 수정해주시기 바랍니다.

SeoJin0513 committed:
Update rules.md

## Summary
- rules.md 파일 내용 수정

## Description
- rules.md 파일 내용 수정
    - 게임 진행 방법에 대해 수정
    - 점수의 획득 조건 내용 추가 및 수정

SeoJin0513 committed:
Update rules.md

SeoJin0513 reviewed (commented):
## Summary
- rules.md 파일 내용 수정

## Description
- rules.md 파일 내용 수정
    - 게임 진행 방법에 대해 수정
    - 점수의 획득 조건 내용 추가 및 수정
    - 오타 수정

kimch0612 reviewed (approved):
LGTM! 수고하셨습니다.
